HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/GiniHealth+PaymentComponentsStringsProvider.swift

```diff
@@ -57,27 +57,27 @@ extension GiniHealth: PaymentComponentsStringsProvider {
                                                                         comment: "Text for tip information label"),
             moreInformationNotePattern: NSLocalizedStringPreferredFormat("gini.health.paymentcomponent.install.app.bottom.sheet.notes.description",
                                                                          comment: "Text for notes information label"),
-            continueLabelText: NSLocalizedStringPreferredFormat("gini.health.paymentcomponent.install.app.bottom.sheet.continue.button.text",
+            continueLabelText: NSLocalizedStringPreferredFormat("c",
```

Comment:
[Clarification]: @razvancapra did you remove the title here by purpose?

Reply:
No, there was a wrong deletion. Fixed!